tests/tt_metal/tt_metal/unit_tests_fast_dispatch/command_queue/test_EnqueueTrace.cpp

```diff
@@ -91,23 +91,24 @@ constexpr bool kBlocking = true;
 constexpr bool kNonBlocking = false;
 vector<bool> blocking_flags = {kBlocking, kNonBlocking};
 
-TEST_F(CommandQueueFixture, InstantiateTraceSanity) {
-    CommandQueue& command_queue = this->device_->command_queue();
+TEST(TraceTests, InstantiateTraceSanity) {
+    Device* device = tt::tt_metal::CreateDevice(0, 1, 0, 2048);
+    CommandQueue& command_queue = device->command_queue();
```

Comment:
For the fixture changes where the device is now created in the test instead of fixture, should we put them in try excepts so that we can always close device on failure?